src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -641,6 +644,7 @@ public void windowClosing(WindowEvent e) {
             bp.updateSearchManager();
             // Set correct enabled state for Back and Forward actions:
             bp.setBackAndForwardEnabledState();
+            bp.getUndoManager().postUndoRedoEvent();
```

Comment:
To set the correct enable/disable state on the panel. Maybe a bit hacky,
but the alternative is to define another event type.